.vscode/launch.json

```diff
@@ -192,6 +192,29 @@
         "{env:DATA_ROOT}/rme-scrape",
       ]
     },
+
+     {
+      "name": "RME to Athena",
+      "type": "debugpy",
+      "request": "launch",
+      "program": "${workspaceFolder}/scripts/geo/rme_to_athena.py",
+      "cwd": "${workspaceFolder}",
+      "console": "integratedTerminal",
+      "envFile": "${workspaceFolder}/.env",
+      "env": {
+        "PYTHONPATH": "${workspaceFolder}"
+      },
+      "args": [
+        "production",
+        "/opt/homebrew/lib/mod_spatialite.8.dylib",
```

Comment:
@philipbaileynar can you please include something in a readme somewhere (if you haven't already) for people who might stumble on this on how to get this file and how to use it? Just a couple lines would do I think.

Reply:
Good idea! Done!

https://github.com/Riverscapes/data-exchange-scripts?tab=readme-ov-file#spatialite